### ISSUE TITLE
Send Certified Email (PEC, RFC 6109)

### ISSUE THREAD

**Issue by matteoraggi** [sympa-community/sympa#1427]
Hello, how to send Certified email?
Here what the certified emails are:
https://en.wikipedia.org/wiki/Certified_email
In Italy the Certified Email are called PEC (Posta Elettronica Certificata).

**Comment by racke** [comment]
How is that related to Sympa?

**Comment by ikedas** [comment]
I guess the intentions is that:

I found [a discussion](https://www.condominioweb.com/forum/quesito/invio-pec-a-pi%C3%B9-persone-152824/) about software for management of the condominium (in Italian).  In this, there is a discussion about how to send the PEC e-mail to all condo residents, and also about whether the mailing list is an option.

I guess that, In conclusion, PEC e-mail could be sent through a mailing list, but in that case the sender would not receive a delivery receipt.  Because the mailing list server rewrites the SMTP envelope which is used to determine the identity of the PEC e-mail.

In other words, PEC e-mail may be sent through the mailing list, but it will be treated as just a S/MIME-signed e-mail: Recipients may receive the e-mail, but it cannot be used to guarantee on sender identity.
